evidence/characters/electro/sara.md

```diff
@@ -203,6 +203,29 @@ Number of A4 procs (from Sara E with C2) it would take to completely fill Burst
 **Significance:**  
 Decorum's wording implies that you need to hit 100% ER increments to restore more Energy, but this is not true. Sara restores Energy directly based on her ER, so for every 1% of ER, you generate 0.012 flat Energy from Decorum.
 
+#### Pre-A4 EMC Skill ER Buff with Sara A4
```

Comment:
despite this being sara a4, it still should go to Synergies/Interactions

Reply:
moved